application/src/main/java/org/opentripplanner/updater/spi/PollingGraphUpdater.java

```diff
@@ -40,7 +41,15 @@ public abstract class PollingGraphUpdater implements GraphUpdater {
   /**
    * Parent update manager. Is used to execute graph writer runnables.
    */
-  protected WriteToGraphCallback saveResultOnGraph;
+  private WriteToGraphCallback saveResultOnGraph;
+
+  /**
+   * Handle on the task posted during the previous polling execution.
```

Comment:
I don't think the expression "handle on" is idiomatic, particularly without an article. It might be mistaken for an imperative verb. I suggest we adapt the language in ExecutorService.submit Javadoc: "a Future representing pending completion of the task".
```suggestion
   * A Future representing pending completion of most recently submitted task.
```

Reply:
Done